docroot/themes/humsci/humsci_basic/src/scss/components/_vertical-linked-card.scss

```diff
@@ -229,7 +229,7 @@
 .hb-stretch-vertical-linked-cards {
   height: 100%;
 
-  .field-hs-collection-items .ptype-hs-postcard {
+  & > div > div > div > div {
     display: flex;
   }
 }
```

Comment:
Sorry, but this doesn't work. You're assuming a specific number of divs, this is probably less generic than the previous implementation.

Why do we need the `display: flex` here? I tried removing it and the english site works well.

Reply:
@cienvaras I don't know why but at the beginning it wasn't working without that. But now that you mentioned, I removed it and tested, and you're right. Thanks for catching it up!